Authentication/login.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+
+    <link rel="stylesheet" href="https://garnish-ui.netlify.app/styles.css">
+    <link rel="stylesheet" href="/Authentication/authentication.css">
+    <link rel="stylesheet" href="/styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+</head>
+<body>
+
+    <header class="header-nav flex">
+        <ul class="navigation navbar-left flex">
+            <li><a href="/index.html" class="nav-pill ">Home</a></li>
+            <li><a href="" class="nav-pill">Products</a></li>
+        </ul>
+
+        <input class="search-bar" type="text" placeholder="Search" />
+
+        <ul class="navigation navbar-right flex">
+            <li>
+                <a href="/Wishlist/wishlist.html" class="nav-pill"><i class="material-icons"> favorite_border</i></a>
+            </li>
+
+            <li>
+                <a href="/Authentication/login.html" class="nav-pill"><i class="material-icons"> account_circle</i></a>
+            </li>
+
+            <li>
+                <a href="/Cart/cart.html" class="nav-pill"><i class="material-icons"> add_shopping_cart</i></a>
+            </li>
+        </ul>
+    </header>
+
+    <div class="page-wrapper flex-page">
```

Comment:
instead of using one div with as a wrapper for the entire page (if that's the case) use div as section wrapper, it would increase your work but would create a better element structure.